### PR TITLE
fix(pugixml): add `includes` argument

### LINKED ISSUE #2350: [Bug]: missing `includes = ["."]` in `pugixml` module
### What happened?

I am trying to use the `pugixml` module. Existing code that uses `#include <pugixml.h>` does not  work. I need to use `#include "pugixml.h"`. This is burdensome for libraries that support Bazel and CMake.

### Version

Development (host) and target OS/architectures:

Linux, many variants.

Output of `bazel --version`:

```
bazel 7.2.1
```

Version of relevant rules from the `WORKSPACE` or `MODULE.bazel` file:

Language(s) and/or frameworks involved:

C++

### How to reproduce

```shell
# Untar the attached tarball and then run:

bazel build :hello
```

### Any other information?

Use [bzlmod-pugixml.tar.gz](https://github.com/user-attachments/files/16069769/bzlmod-pugixml.tar.gz) to test:

```
INFO: Analyzed target //:hello (85 packages loaded, 412 targets configured).
ERROR: /usr/local/google/home/coryan/bzlmod-pugixml/BUILD.bazel:5:8: Compiling hello.cc failed: (Exit 1): gcc failed: error executing CppCompile command (from target //:hello) /usr/bin/gcc -U_FORTIFY_SOURCE -fstack-protector -Wall -Wunused-but-set-parameter -Wno-free-nonheap-object -fno-omit-frame-pointer '-std=c++14' -MD -MF ... (remaining 25 arguments skipped)

Use --sandbox_debug to see verbose messages from the sandbox and retain the sandbox build root for debugging
hello.cc:2:10: fatal error: pugixml.h: No such file or directory
    2 | #include <pugixml.h>
      |          ^~~~~~~~~~~
compilation terminated.
Target //:hello failed to build
Use --verbose_failures to see the command lines of failed build steps.
INFO: Elapsed time: 2.486s, Critical Path: 0.10s
INFO: 8 processes: 8 internal.
ERROR: Build did NOT complete successfully
```

### PR DESCRIPTION
**Pull request by coryan**
Fixes #2350

### CODE REVIEW

**Comment by bazel-io** [comment]
Hello @bazelbuild/bcr-maintainers, modules without existing maintainers (pugixml) have been updated in this PR. Please review the changes.

**Comment by meteorcloudy** [comment]
BCR has an "Add-only" policy that ensures checked-in modules are never modified. Can you please send this as a new module version named "1.14.bcr.1? See https://github.com/bazelbuild/bazel-central-registry/tree/main/docs#versions-format

File: modules/pugixml/1.14.bcr.1/presubmit.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  platform:
+  - debian10
+  - ubuntu2004
+  - macos
+  - macos_arm64
+  - windows
+tasks:
+  verify_targets:
+    name: Verify build targets
+    bazel: ${{ bazel }}
```
Comment (meteorcloudy) [inline modules/pugixml/1.14.bcr.1/presubmit.yml:11]: bazel is not defined in the matrix

**Review by meteorcloudy — CHANGES_REQUESTED** [review]

**Comment by coryan** [comment]
PTAL

**Review by meteorcloudy — APPROVED** [review]

**Pull request merged**